src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/NavigationTests.java

```diff
@@ -141,4 +146,33 @@ public void MercuryCuratedNavigationTest_004_navigateThroughDifferentUrl() {
     section.isAlertNotificationVisible();
     Assertion.assertUrlEqualToCurrentUrl(driver, expectedUrl);
   }
+
+  @Test(groups = "MercuryCuratedNavigationTest_005")
+  @Execute(onWikia = MercuryWikis.MERCURY_CC)
```

Comment:
Because curated content is enabled on this wiki, and this way it is assured that the functionality is tested where it can be tested.